shim/build.rs

```diff
@@ -43,20 +43,21 @@ fn expect_release_lib() -> String {
         _ => panic!("unsupported target {}", target.triple),
     };
     let mut dir = PathBuf::from("libturbo");
-    // format is ${BUILD_ID}_${OS}_${ARCH}. Build id is, for goreleaser reasons, turbo-${OS}
+    // format is ${BUILD_ID}_${OS}_${ARCH}. Build id is, for goreleaser reasons,
+    // turbo-${OS}
     dir.push(format!("turbo-{platform}_{platform}_{arch}"));
     dir.push("lib");
     dir.to_string_lossy().to_string()
 }
 
 fn build_debug_libturbo() -> String {
-    let cli_path = "../cli";
+    let cli_path = env::current_dir().unwrap().parent().unwrap().join("cli");
```

Comment:
Should this be using [CARGO_WORKSPACE_DIR](https://github.com/vercel/turbo/blob/1628df924e8e4d57046dbb00f325769cf808ec6d/.cargo/config.toml#L2) instead?